cli/common/utils_test.go

```diff
@@ -115,39 +114,173 @@ func TestGetInstancesFromArgs(t *testing.T) {
+	instances, err = GetInstancesFromArgs(args, projectName)
 	assert.EqualError(err, appNameSpecifiedError)
 
 	args = []string{"myapp", "instance-1"}
-	instances, err = GetInstancesFromArgs(args, ctx)
+	instances, err = GetInstancesFromArgs(args, projectName)
 	assert.EqualError(err, appNameSpecifiedError)
 }
+
+func TestCorrectDependencyParsing(t *testing.T) {
+	t.Parallel()
+
+	assert := assert.New(t)
+
+	rawDeps := []string{
+		"dependency_01 > 1.2, <= 4",
+		"dependency_02 < 7,>=1.5",
+		"dependency_03==2.8 // comment here",
+		"// comment",
+		"	dependency_04   <= 5.2   ",
+		"",
+		"\tdependency_05>=2.4,<=5.1",
+		"dependency_06 =15",
+	}
+
+	deps, err := ParseDependencies(rawDeps)
+
+	assert.Equal(nil, err)
+	assert.Equal(deps, PackDependencies{
+		{
+			Name: "dependency_01",
+			Relations: []DepRelation{
+				{
+					Relation: ">",
+					Version:  "1.2",
+				},
+				{
+					Relation: "<=",
+					Version:  "4",
+				},
+			}},
+		{
+			Name: "dependency_02",
+			Relations: []DepRelation{
+				{
+					Relation: "<",
+					Version:  "7",
+				},
+				{
+					Relation: ">=",
+					Version:  "1.5",
+				},
+			}},
+		{
+			Name: "dependency_03",
+			Relations: []DepRelation{
+				{
+					Relation: "==",
+					Version:  "2.8",
+				},
+			}},
+		{
+			Name: "dependency_04",
+			Relations: []DepRelation{
+				{
+					Relation: "<=",
+					Version:  "5.2",
+				},
+			},
+		},
+		{
+			Name: "dependency_05",
+			Relations: []DepRelation{
+				{
+					Relation: ">=",
+					Version:  "2.4",
+				},
+				{
+					Relation: "<=",
+					Version:  "5.1",
+				},
+			}},
+		{
+			Name: "dependency_06",
+			Relations: []DepRelation{
+				{
+					Relation: "=",
+					Version:  "15",
+				},
+			},
+		},
+	})
+}
+
+func TestInvalidDependencyParsing(t *testing.T) {
+	t.Parallel()
+
+	assert := assert.New(t)
+	rawDeps := []string{
+		"dependency_01 > 1.2, <= 4",
+		"dep01 >= 14, <= 25, > 14",
+	}
+
+	deps, err := ParseDependencies(rawDeps)
+	assert.Equal(
+		"Error during parse dependencies file: 1:19: unexpected token \",\". Trying "+
+			"to parse: dep01 >= 14, <= 25, > 14",
+		err.Error(),
+	)
+	assert.Equal(PackDependencies(nil), deps)
+
+	rawDeps = []string{
+		"broke broke broke broke broke broke broke",
+	}
+
+	deps, err = ParseDependencies(rawDeps)
+	assert.Equal(
+		"Error during parse dependencies file: 1:7: unexpected token \"broke\". Trying "+
+			"to parse: broke broke broke broke broke broke broke",
+		err.Error(),
+	)
+	assert.Equal(PackDependencies(nil), deps)
+
+	rawDeps = []string{
+		"dependency , ,",
+	}
+
+	deps, err = ParseDependencies(rawDeps)
+
+	assert.Equal("Error during parse dependencies file: 1:12: unexpected token \",\". Trying "+
+		"to parse: dependency , ,", err.Error())
+	assert.Equal(PackDependencies(nil), deps)
+
+	rawDeps = []string{
+		"dependency >= 3.2, < ",
+	}
+
+	deps, err = ParseDependencies(rawDeps)
+
+	assert.Equal("Error during parse dependencies file: 1:21: unexpected token \"<EOF>\" "+
```

Comment:
I think that `"Error during parse dependencies file" in err.Error()` check is enough.